client/src/clientRoutes/__mocks__/axios.js

```diff
@@ -1,4 +1,26 @@
+
+import exampleData from '../../relatedProducts/newExampleData.js'
+
+
+
+let newStyle = (id) => {
```

Comment:
this function was originally used for the mock server request for related products.  However i was able to get it working without so it can be removed